### ISSUE TITLE
Issue facing with nn.functional.sigmoid()

### ISSUE THREAD

**Issue by Tarakes796** [sahilkhose/FloodNet#4]
Hi,
I am trying to implement your model for basic of my project but I am getting below warning:
/usr/local/lib/python3.9/dist-packages/torch/nn/functional.py:1967: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

and also I am getting low mIoU values. What is the reason for this?
